README.md

```diff
@@ -1,44 +1,168 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-# Project Name
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-> Description the project.
+Modify this file to match your project and remove sections that don't apply.
 
+For example: Delete the testing section if the currect project doesn't require testing
+-->
 
-## Built With
+<div align="center">
 
-- Major languages
-- Frameworks
-- Technologies used
+  <img src="mv-readme-logo.png" alt="logo" width="240"  height="auto" />
+  <br/>
 
-## Live Demo (if available)
+  <h3><b>Microverse README Template</b></h3>
 
-[Live Demo Link](https://livedemo.com)
+</div>
 
+<!-- TABLE OF CONTENTS -->
 
-## Getting Started
+# 📗 Table of Contents
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
+<!-- PROJECT DESCRIPTION -->
 
-To get a local copy up and running follow these simple example steps.
+# 📖 [Project name] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[High level description]**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features
```

Comment:
WAs that required by the CS team?

Reply:
It was my own input. Perhaps it's making the template too complicated, as you mentioned [earlier](https://app.asana.com/0/0/1203018001393729/1203075461623943/f).

As I went to update the template it seems to make educational sense to add these extra sections:
- Key features
- Tech stack
- Getting started

I know it'll take more time from CRs, but these sections do seem important. What do you think?

Reply:
Considering the new CR training, it might actually just be **best to not introduce too many changes at once**. I'll remove these extra sections for now and perhaps we reintroduce it at some later stage when it makes more sense 🙂